ee/conftest.py

```diff
@@ -91,6 +92,14 @@ def reset_clickhouse_tables():
         sync_execute(item)
 
 
+def run_additional_setup_ops():
```

Comment:
why do we need to do this, why doesn't the fresh install initial migration add that comment?

Reply:
Because on the test environment we're not actually running the migrations - annoyingly we're setting up the DB independently

Reply:
`conftest` is just a `pytest` thing, you'll see this file runs all the table creation statements and truncates tables in between

Reply:
Oh the test just got way less useful :(

Reply:
it did